packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/double-close-template/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "warnings": [
+        {
+            "code": 1148,
+            "message": "LWC1148: Invalid HTML syntax: end-tag-without-matching-open-element. This will not be supported in future versions of LWC. For more information, please visit https://html.spec.whatwg.org/multipage/parsing.html#parse-error-end-tag-without-matching-open-element",
```

Comment:
@nolanlawson should this jump to a specific section of the HTML Standard? I'm testing this on the lightning-language-server side, and am finding that it doesn't anchor anywhere.

Reply:
@randi274 You're right, this link is broken. I can't find anything in the Interact Archive that corresponds to it.

What it's actually referring to is [this parse5 error](https://github.com/inikulin/parse5/blob/8e22fe40d5774e2f2d5eb9be230f4f78d60e780d/packages/parse5/lib/common/error-codes.ts#L62). I think what's happening here is that we assume there is an error code in the HTML5 spec with the same fragment link as the parse5 error name, but since HTML5 treats this as a recoverable error state, it isn't actually in the HTML5 spec. We should fix this.

Reply:
Opened a bug: https://github.com/salesforce/lwc/issues/2837